main.md

```diff
@@ -460,17 +462,18 @@ grant_type=authorization_code
 !---
 Figure: Token Request for a DPoP sender-constrained token using an authorization code {#token-request-code}
 
-The `DPoP` HTTP header MUST contain a valid DPoP proof JWT.
 If the DPoP proof is invalid, the authorization server issues an error 
 response per Section 5.2 of [@!RFC6749] with `invalid_dpop_proof` as the
 value of the `error` parameter. 
 
-To sender-constrain the access token, after checking the validity of the
-DPoP proof, the authorization server associates the issued access token with the
-public key from the DPoP proof, which can be accomplished as described in (#Confirmation).
-A `token_type` of `DPoP` MUST be included in the access token
-response to signal to the client that the access token was bound to
-its DPoP key and can be used as described in (#http-auth-scheme). 
+If the DPoP proof is valid, the access token response
```

Comment:
This changes the meaning by putting a MUST on the DPoP proof being valid. There were a number of folks in the WG that wanted the ability to bind a RT only. Changing that is more than an editorial improvement.

Reply:
@b---c :thinking: can I use an invalid DPoP proof? What happens in this case?